### DISCUSSION TITLE
How to download all user actions in a private organization through API?

### DISCUSSION THREAD

**Discussion by victor-kwok** [community/community#42192] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I tried

https://api.github.com/users/victor-kwok/events/org/{our_orgs}’
and only 45 events came back. we have hundreds of events daily as a company. 

Any suggestion?

https://docs.github.com/en/rest/activity/events?apiVersion=2022-11-28#list-organization-events-for-the-authenticated-user

this only works for my own events

That API end point seems to only work for my own events. Do we have any way to do this for all users in the organization scriptomatically? It feels weird if I could view it, individually, on the UI but cannot do the same through the API with my account.

**Comment by byrneh** [comment]
A GitHub Enterprise Cloud subscription may be  needed to achieve this. It has an audit events api feature
